### PR TITLE
adding active tab to breadcrumb

### PR DESCRIPTION
**Pull request by heper**
closely review L186 - L188 (commented out some code, that is possibly used in some packages)

### CODE REVIEW

File: src/usr/local/www/pkg.php [hunk 1]
```diff
@@ -183,9 +183,9 @@ function domTT_title($title_msg) {
 
 // Breadcrumb
 if ($pkg['title'] != "") {
-	if (!$only_edit) {
- 		$pkg['title'] = $pkg['title'] . '/Edit';
-	}
+	/*if (!$only_edit) {						// Is any package still making use of this?? Is this something that is still wanted, considering the breadcrumb policy https://redmine.pfsense.org/issues/5527
```
Comment (sbeaver-netgate) [inline src/usr/local/www/pkg.php]: Yes. There are packages that still use edit_only.

Comment (heper) [inline src/usr/local/www/pkg.php]: @sbeaver-netgate $only_edit is not set anywhere in pkg.php.
this means that, this code will always add '/Edit' to the title. Is that the way it is supposed to be then ?

Comment (sbeaver-netgate) [inline src/usr/local/www/pkg.php]: Perhaps I was thinking on pkg_edit.php

I'll merge this and we can look out for any issues then. Thanks.

**Comment by heper** [comment]
@sbeaver-netgate 
i think some of the code from pkg_edit.php got copied to pkg.php a couple of days ago:
see: https://github.com/pfsense/pfsense/commit/8cfdeb2ef63e283fad931f220d1f1930651097d4

**Comment by sbeaver-netgate** [comment]
Ah yes. That explains it. Thanks!

Steve
--
Stephen Beaver | +1 (512) 646-4100;239 | sbeaver@netgate.com | www.netgate.com

**Pull request merged**